gherkin/gherkin-languages.json

```diff
@@ -2566,7 +2566,7 @@
     "name": "Polish",
     "native": "polski",
     "rule": [
-      "Rule"
+      "Zasada"
```

Comment:
Wouldn't `Reguła` be more suitable in this context?

Reply:
I don't know polish
But I know that you Gherkin allows to have several words for a same key.

Maybe you may have both translations: "Zasada" and "Reguła"?

Reply:
@l3l0 do you have the possibility to add the second translation as suggested by @jakzal?